apps/web/app/(preview)/style/globals.css

```diff
@@ -5,6 +5,37 @@
 @tailwind utilities;
 
 @layer base {
+  :root {
+    --button-font-family: var(--font-family-body);
+
+    --button-primary-background: hsl(var(--primary));
+    --button-primary-background-hover: color-mix(in oklab, hsl(var(--primary)), white 75%);
+    --button-primary-foreground: hsl(var(--foreground));
+    --button-primary-foreground-hover: hsl(var(--foreground));
+    --button-primary-border: hsl(var(--primary));
+    --button-primary-focus: hsl(var(--foreground));
+
+    --button-secondary-background: hsl(var(--foreground));
+    --button-secondary-background-hover: hsl(var(--background));
+    --button-secondary-foreground: hsl(var(--background));
+    --button-secondary-foreground-hover: hsl(var(--foreground));
+    --button-secondary-border: hsl(var(--foreground));
+    --button-secondary-focus: hsl(var(--foreground));
+
+    --button-tertiary-background: hsl(var(--background));
+    --button-tertiary-background-hover: hsl(var(--contrast-100));
+    --button-tertiary-foreground: hsl(var(--foreground));
+    --button-tertiary-foreground-hover: hsl(var(--foreground));
+    --button-tertiary-border: hsl(var(--contrast-200));
+    --button-tertiary-focus: hsl(var(--foreground));
+
+    --button-ghost-background: transparent;
+    --button-ghost-background-hover: hsl(var(--foreground) / 5%);
+    --button-ghost-foreground: hsl(var(--foreground));
+    --button-ghost-foreground-hover: hsl(var(--foreground));
+    --button-ghost-border: transparent;
+    --button-ghost-focus: hsl(var(--foreground));
+  }
```

Comment:
Don't we want to apply this via the Soul schema and the CSS that's injected by that specific Soul brand?

Reply:
Nope because the Soul brand schemas should be just updating the underlying design tokens that power this theme!